src/AzureAppConfigurationImpl.ts

```diff
@@ -29,20 +36,41 @@ export class AzureAppConfigurationImpl extends Map<string, unknown> implements A
         super();
         // Enable request tracing if not opt-out
         this.requestTracingEnabled = requestTracingEnabled();
-        if (this.requestTracingEnabled) {
-            this.enableRequestTracing();
-        }
 
         if (options?.trimKeyPrefixes) {
             this.sortedTrimKeyPrefixes = [...options.trimKeyPrefixes].sort((a, b) => b.localeCompare(a));

```

Comment:
why do we need to `sort`>

Reply:
The purpose is to trim the longest prefix first, if one is substring of another. E.g. 
prefixes = ["app.", "app.settings."], we expect "app.settings.message" to be trimmed as "message".